django/santropolFeast/member/forms.py

```diff
@@ -80,6 +81,306 @@ class ClientRestrictionsInformation(forms.Form):
         help_text=_('By default, the client meal status is Pending.'),
         required=False,
     )
+    size_monday = forms.ChoiceField(
+        choices=SIZE_CHOICES,
+        widget=forms.Select(
+            attrs={'class': 'ui dropdown'}
+        )
+    )
+    size_monday.required = False
```

Comment:
You could pass "required=False"  to the Field constructor directly? something like this: 
`size_monday = forms.ChoiceField(choices..., required=False)`
The same could be done with the other fields.
[https://docs.djangoproject.com/en/1.9/ref/forms/fields/#required](url)